_run/lite/Makefile

```diff
@@ -87,10 +96,16 @@ bid-close:
 order-close:
 	$(akashctl) tx $(KEY_OPTS) $(CHAIN_OPTS) market order-close -y \
 	--owner $(MAIN_ADDR) \
+	--from provider \
+	--dseq "$(DSEQ)"  \
+	--gseq "$(GSEQ)"
+
+group-close:
+	$(akashctl) tx $(KEY_OPTS) $(CHAIN_OPTS) deployment group close \
+	--owner $(MAIN_ADDR) \
 	--from main \
 	--dseq "$(DSEQ)"  \
 	--gseq "$(GSEQ)"  \
-	--oseq "$(OSEQ)"  
```

Comment:
I think the last line of the command still ends in a `\`

Reply:
https://www.gnu.org/software/make/manual/html_node/Splitting-Recipe-Lines.html#:~:text=5.1.,1%20Splitting%20Recipe%20Lines&text=As%20in%20normal%20makefile%20syntax,be%20invoked%20to%20run%20it.

Makefile docs show the last line doesn't require another newline to be added.

This is the pattern I've always seen:
```
.PHONY: build-k8s
build-k8s: $(K8S_BUILD_DIR)
	@for file in $(K8S_FILES); do \
		mkdir -p `dirname "$(K8S_BUILD_DIR)/$$file"` ; \
		$(SHELL_EXPORT) envsubst <$(K8S_DIR)/$$file >$(K8S_BUILD_DIR)/$$file ;\
	done
```
Last line doesn't need a `\`: https://blog.zikes.me/post/config-k8s-with-make/

Reply:
exactly, the last line shouldn't have a `\`.  it looks like with this change the last line has a `\` - it is

```
   --gseq "$(GSEQ)"  \
```

it only works because there happens to be an empty line below.